test/activation_based/test_online_learning.py

```
@@ -0,0 +1,100 @@
+
+def test_fptt_running_average_starts_from_independent_parameter_snapshots():
+    model = nn.Linear(2, 1)
+    optimizer = torch.optim.SGD(model.parameters(), lr=0.1)
+    snapshots = fptt_online_training_init_w_ra(optimizer)
+    original = [snapshot.clone() for snapshot in snapshots]
+
+    with torch.no_grad():
+        for parameter in model.parameters():
+            parameter.add_(1.0)
+
+    for snapshot, expected, parameter in zip(snapshots, original, model.parameters()):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)